QModManager/LanguageHelper.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Oculus.Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+
+namespace QModManager
+{
+    public static class LanguageHelper
+    {
+        internal static Dictionary<string, string> strings;
+        internal static Dictionary<string, string> backupStrings;
+        internal static string language;
+
+        public static Location Contains(string key)
+        {
+            if (strings.ContainsKey(key)) return Location.Main;
+            if (backupStrings.ContainsKey(key)) return Location.Backup;
+            return Location.Missing;
+        }
+        public static string Get(string key)
+        {
+            if (string.IsNullOrEmpty(key)) return null;
+            if (strings.ContainsKey(key)) return strings[key];
+            if (backupStrings.ContainsKey(key)) return backupStrings[key];
+            return null;
```

Comment:
If this is text that is going to be printed on screen,  
it may be best to return the `key` to be used as the value instead of `null`.

This way, even in the event of an error, there will still be some intelligible text.

Reply:
Agreed. Will change in the next commit

Reply:
Addressed in commit 664d768